assignments/PaintLab8.html

```diff
@@ -0,0 +1,162 @@
+<!DOCTYPE html>
+<html lang="en">
+
+    <style>
+        table, th, td {
+          table-layout: fixed;
+          border-collapse: collapse;
+          border: 1px solid black;
+        }
+
+        th {
+            padding: .15%;
+        }
+
+        td {
+            padding: .15%;
+            width: 50px;
+            height: 50px;
+            
+        }
+
+        button { 
+            width: 100%; 
+            height: 100%;
+        } 
+    </style>
+
+<head>
+    <title>Paint a Picture - Lab 8</title>
+</head>
+
+<body>
+
+    <div>
+        <table>
+            <tr>
+                <th colspan="8">Click a box to paint a picture</th>
+            </tr>
+            <tr>
+                <td><button type="button"></button></td>
+                <td><button type="button"></button></td>
```

Comment:
For future reference: you would probably want to generate the table cells and their buttons dynamically via JavaScript code rather than copy/pasting all them in the HTML itself.  With the cells here in HTML, if you ever wanted to change how they're structured or what they contain you would have to update every line and this is tedious and error-prone.